pkg/setup.cfg

```diff
@@ -4,7 +4,7 @@ name = gempython_gemplotting
 author = GEM Online Systems Group
 author_email = cms-gem-online-sw@cern.ch
 summary = __summary__
-license = MIT
+license = GNU General Public License v3 (GPLv3)
```

Comment:
License changes *must* be discussed

Reply:
This is not a license change, this is reflecting the fact that the current license *is* GPLv3. If you want to contact all copyright holders, fine; but until you get answers the license remains GPLv3.